design-patterns/gc-ministers/gc-ministers-en.html

```diff
@@ -0,0 +1,144 @@
+---
+title: Minister or institutional head pattern
+description: Documentation and working example for the minister or institutional head pattern 
+language: en
+altLangPage: gc-ministers-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
+	<span class="wb-prettify all-pre hide"></span>
+	<p>The Minister or institutional head pattern identifies the relevant Minister(s) or head(s) for a department, agency or organization. It provides links to the official bio for an institution’s minister(s), associate minister(s), or institutional head, and may also include a headshot or links to mandate letters and briefing materials.</p>	
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+	<h2 id="before-start">Before you start</h2>
+	<p>The Minister or institutional head pattern is mandatory on an institutional landing page. Consult the Canada.ca Design system guidance for using this component.</p>
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+	</ul>
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+	<p>There is <span>no accessibility guidance</span> defined.</p>
+	<h2 id="variations">Variations</h2>
+	<p>The following coded example is the default layout. When you are implementing your instance, you can omit any of the following:</p>
+	<ul>
+		<li>image</li>
+		<li>mandate letter link</li>
+		<li>briefing book link</li>
+	</ul>
+	<h3>Minister or institutional head pattern</h3>
+<div class="container">
+	<div class="row">
+		<div class="mrgn-bttm-lg wb-eqht col-md-12">
+			<h2 class="h3 mrgn-tp-md">Minister(s)</h2>
+			<div class="row">
+				<div class="col-md-6">
+					<div class="row">
+						<figure>
+							<figcaption class="pull-right col-md-7 col-xs-12 mrgn-tp-md bold"><a href="">The Honourable [Minister name]</a>
+							</figcaption>
+							<div class="col-md-5 col-xs-12">
+								<img src="./img/minister-200x250.jpg" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
```

Comment:
This image is cannot be found. Please add it to the PR and update the src path or reference another image in the project. You can also use a placeholder. For example: https://placehold.it/200x250/000000/ffffff?text=Minister. Do this for both images.

Reply:
Added the image, there is only 1 used twice, since they are placeholders